File: prql-php/src/MessageKind.php
<?php

declare(strict_types=1);

namespace Prql\Compiler;

/**
 * Compile message kind. Currently only Error is implemented.
 */
enum MessageKind
{
    /**
     * Error message.
     */
    case Error;
    /**
     * Warning message.
     */
    case Warning;
    /**
     * Lint message.
     */
    case Lint;
}

File: prqlc/bindings/dotnet/PrqlCompiler/MessageKind.cs
using System;

namespace Prql.Compiler;

/// <summary>
/// Compile message kind. Currently only Error is implemented.
/// </summary>
[Serializable]
public enum MessageKind
{
    /// <summary>
    /// Error message.
    /// </summary>
    Error,
    /// <summary>
    /// Warning message.
    /// </summary>
    Warning,
    /// <summary>
    /// Lint message.
    /// </summary>
    Lint
}
